docs/proposals/Gallery/README.md

```diff
@@ -0,0 +1,58 @@
+# Component Name
+
+The `Gallery` component will accept a list of attachments for images and display
+them as thumbnails. This would also have an option to show the images in a
+LightBox component on click of those images.
+
+## Accessibility
+
+Assistive technology should be able to see the images and know the image name
+based on the `name` property of each attachment.
+
+## Responsiveness & Mobile
+
+This component show maximum number of images possible in a line and show the
```

Comment:
I have some questions about what is being described here. What does "maximum number possible" mean?
Do we want the component to always go full width in the container it appears in, and then truncate at one line of thumbnails? What sizes of items do we expect to put in the gallery? (I think some designer considerations might help here.)